assets/js/components/user-input/util/constants.js

```diff
@@ -21,18 +21,14 @@
  */
 import { __ } from '@wordpress/i18n';
 
-export const USER_INPUT_QUESTION_ROLE = 'role';
 export const USER_INPUT_QUESTION_POST_FREQUENCY = 'postFrequency';
-export const USER_INPUT_QUESTION_GOALS = 'goals';
-export const USER_INPUT_QUESTION_HELP_NEEDED = 'helpNeeded';
-export const USER_INPUT_QUESTION_SEARCH_TERMS = 'searchTerms';
+export const USER_INPUT_QUESTIONS_PURPOSE = 'goals';
+export const USER_INPUT_QUESTIONS_GOALS = 'helpNeeded';
```

Comment:
The value of these constants should be updated too, otherwise, they would be saved to the wrong key in the database, i.e.
1. `USER_INPUT_QUESTIONS_PURPOSE` => `purpose`
2. `USER_INPUT_QUESTIONS_GOALS` => `goals`

Also, it would be nice to order them based on their visibility.